components/staging/staging.html

```diff
@@ -20,9 +20,17 @@
           </span>
           <span class="commit-message-title-counter" data-bind="text: commitMessageTitleCount"/>
         </div>
-        <button class="btn btn-primary btn-lg commit-btn" data-bind="click: commit, visible: isStageValid, enable: !commitValidationError()">
-          Commit
-        </button>
+        <div class="btn-group commit-grp" data-bind="visible: isStageValid">
+          <button class="btn btn-primary btn-lg commit-btn" data-bind="click: commit, enable: !commitValidationError()">
+            Commit
+          </button>
+          <button type="button" class="btn btn-primary btn-lg dropdown-toggle" data-toggle="dropdown">
```

Comment:
Hmm.. not sure if this button should also be disabled. It looks strange to have the commit button disabled but the dropdown is enabled which then shows only disabled entries.

![image](https://user-images.githubusercontent.com/4009570/36077106-b23519fe-0f66-11e8-99b3-f17ebb8eec50.png)

See Visual Studio for comparison
![image](https://user-images.githubusercontent.com/4009570/36077093-90da4220-0f66-11e8-8f05-f0136fce3f1e.png)

Reply:
OK. I updated the PR to disable the dropdown toggle too and squashed the commits to have only one.